README.md

```diff
@@ -105,8 +105,10 @@ e.g. `${{steps.setup-beam.outputs.erlang-version}}`
 
 ### Version file
 
-A version file is specified via input `version-file` (e.g.`.tool-versions`). This
-allows not having to use YML input for versions, though the action does check (and
+A version file can be used via inputs `version-file` which specifies the file path (e.g.`.tool-versions`)
+and `version-file-type` which can be set to either `asdf` or `mise`.
```

Comment:
I'd prefer "... and `version-file-type` which can be set to one of the values in table _Mapping between YML and version file formats_" (where there'd be a link to the table).